### DISCUSSION TITLE
npm install failure

### DISCUSSION THREAD

**Discussion by BurrellCreekCommunity** [heremaps/harp.gl#2237] [Q&A]
Hi
npm i @here/harp.gl fails with the message "npm ERR! 404 Not Found - GET https://repo.platform.here.com/artifactory/api/npm/maps-api-for-javascript/@here%2fharp.gl - not_found"

Is there something that I am missing?

Thanks

**Comment by nzjony** [comment]
@BurrellCreekCommunity , thanks for the discussion.

Do you have a `.npmrc` in your root directory? What does it say?

The @here/harp.gl package is available https://www.npmjs.com/package/@here/harp.gl
